config/env.yml

```diff
@@ -0,0 +1,3 @@
+CONTACT: 0900 786011
```

Comment:
this file should be removed. and not added in commit

Reply:
okay